test/views/pages/employmentsupport/RemoveClaimPageViewSpec.scala

```diff
@@ -74,6 +73,8 @@ class RemoveClaimPageViewSpec extends ViewUnitTest {
     val doNotRemoveLinkHiddenText: String
 
     val no: String
+
+    def expectedEndDateQuestionRowKey(taxYear: Int, startDate: LocalDate): String
```

Comment:
Please make this test the same as the jobseekers test. They should mirror each other. You have done some refactoring on the one, but not the other